superbuild/projects_modules/medInria.cmake

```diff
@@ -54,7 +54,7 @@ if (NOT USE_SYSTEM_${ep})
 # set compilation flags
 if (UNIX)
   set(${ep}_c_flags "${${ep}_c_flags} -Wall")
-  set(${ep}_cxx_flags "${${ep}_cxx_flags} -Wall")
+  set(${ep}_cxx_flags "${${ep}_cxx_flags} -Wall -Wno-unknown-pragmas")
```

Comment:
Remember this line modification is here only because of dtk.
It must be removed on 4.1
an equivalent of this fix for Windows would be fine.

Reply:
oki, i do that as soon as possible

Reply:
@Florent2305 : do you have these pragmas warning messages? If yes, is this parameter working to solve the problem?